c9/pwb.py

```
@@ -0,0 +1,176 @@
+
+
+def execute():
+    """Parse arguments, extract filename and run the script.
+
+    .. versionadded:: 7.0
+       renamed from :func:`main`
+    """
+    filename, script_args = handle_args(*sys.argv)
+
+    if not filename:
+        warn("No filename given")
```

Ruff (0.15.12)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)